<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CBlobFeaturesSupplierGuiComp</class>
 <widget class="QWidget" name="CBlobFeaturesSupplierGuiComp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>420</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="TestButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Test</string>
       </property>
       <property name="icon">
        <iconset resource="../../../Acf/Include/iqtgui/iqtgui.qrc">
         <normaloff>:/Icons/Play</normaloff>:/Icons/Play</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="AutoTestButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../Acf/Include/iqtgui/iqtgui.qrc">
         <normaloff>:/Icons/AutoUpdate</normaloff>:/Icons/AutoUpdate</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="ParamsGB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Parameters</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="ParamsFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="LoadParamsButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Load...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SaveParamsButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Save...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="IntermediateWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Results</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTreeWidget" name="ResultsList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
          <property name="toolTip">
           <string>ID of the blob</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Position</string>
          </property>
          <property name="toolTip">
           <string>Position of the found blob</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Area</string>
          </property>
          <property name="toolTip">
           <string>Area of the blob</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Diameter</string>
          </property>
          <property name="toolTip">
           <string>Diameter of the blob</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Circularity</string>
          </property>
          <property name="toolTip">
           <string>Circularity of the blob</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Perimeter</string>
          </property>
          <property name="toolTip">
           <string>Perimeter of the blob (total length of the contour)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Angle</string>
          </property>
          <property name="toolTip">
           <string>Angle of found model [Â°]</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>TestButton</tabstop>
  <tabstop>AutoTestButton</tabstop>
  <tabstop>LoadParamsButton</tabstop>
  <tabstop>SaveParamsButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../../Acf/Include/iqtgui/iqtgui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>AutoTestButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>TestButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
